src/i18n/i18n.js

```diff
@@ -0,0 +1,41 @@
+import i18n from "i18next";
+import LanguageDetector from "i18next-browser-languagedetector";
+
+i18n.use(LanguageDetector).init({
+  // we init with resources
+  resources: {
+    en: {
```

Comment:
This approach is ok for now, maybe add a @TODO to split the translations text in different files ;)